lib/hook.js

```diff
@@ -46,6 +46,22 @@ function Hook() {
 	};
 }
 
+function findInDir (dir, include = /\.js$/, exclude = /^[._]/, fileList = []) {
+	const files = fs.readdirSync(dir);
+	files.forEach((file) => {
+		const filePath = path.join(dir, file);
+		const fileStat = fs.lstatSync(filePath);
```

Comment:
Why `fs.lstatSync()` instead of `fs.statSync()`? You only really need `fs.lstatSync()` if you need to know if a file is a symlink. In this case, you don't care if it's a symlink, you just want to walk it.

Reply:
changed to statSync()